src/client/containers/CartPage/CartPage.Container.js

```diff
@@ -67,7 +67,7 @@ const CartPageContainer = ({ isAuthenticated }) => {
     changeProductQuantity(productId, 0);
   };
   return (
-    <div>
+    <div className="cartPage">
```

Comment:
Another pattern for naming classes? Maybe stick with the same pattern. 🙂

Reply:
Yes .. I should fix this ..Thanks